ports/esp32/modutime.c

```diff
@@ -29,9 +30,74 @@
 #include <stdio.h>
 #include <string.h>
 #include <sys/time.h>
+#include <py/nlr.h>
+#include <py/smallint.h>
+#include "timeutils.h"
 
 #include "extmod/utime_mphal.h"
 
+/// \module time - time related functions
+///
+/// The `time` module provides functions for getting the current time and date,
+/// and for sleeping.
+
+/// \function localtime([secs])
+/// Convert a time expressed in seconds since Jan 1, 2000 into an 8-tuple which
+/// contains: (year, month, mday, hour, minute, second, weekday, yearday)
+/// If secs is not provided or None, then the current time from the RTC is used.
+/// year includes the century (for example 2014)
+/// month   is 1-12
+/// mday    is 1-31
+/// hour    is 0-23
+/// minute  is 0-59
+/// second  is 0-59
+/// weekday is 0-6 for Mon-Sun.
+/// yearday is 1-366
```

Comment:
No need to copy across these comments.  They are obsolete and the info is provided in the docs.